scripts/test/Muon/plotting_canvas/plotting_canvas_presenter_test.py

```diff
@@ -185,6 +185,78 @@ def test_create_tiled_plot_correctly_handles_empty_key_list(self):
 
         self.view.create_new_plot_canvas.assert_called_once_with(1)
 
+    @mock.patch(
+    'Muon.GUI.Common.plot_widget.plotting_canvas.plotting_canvas_presenter.PlottingCanvasPresenter.'
+    'autoscale_selected_y_axis')
+    def test_handle_autoscale_correctly_handles_one_subplot(self,mock_autoscale_selected_y_axis):
+        self.presenter._options_presenter.check_autoscale_state = mock.Mock()
```

Comment:
we can clean these up by using:
```suggestion
        self.presenter._options_presenter.check_autoscale_state = mock.Mock(return_value = True)
```